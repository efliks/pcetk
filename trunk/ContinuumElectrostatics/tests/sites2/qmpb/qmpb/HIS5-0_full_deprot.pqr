../0/site_HIS_5_full_deprot.pqr