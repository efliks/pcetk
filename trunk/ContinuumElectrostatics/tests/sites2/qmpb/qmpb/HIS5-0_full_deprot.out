Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named HIS5-0_full_deprot
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (4.3172, 3.76156, 5.21288)) 121 2
Finer level:    (0.435, -0.5, 5.728) 101 1
Finer level:    (0.435, -0.5, 5.728) 101 0.5
Finest level:   (0.435, -0.5, 5.728) 101 0.25


Self energy of HIS5-0_full_deprot in three dielectric environment = 367.733
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of HIS5-0_full_deprot with background charge set in three dielectric environment = -0.141371
(probably in kcal/mole)



Interaction energies of HIS5-0_full_deprot with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 519.06
0   1 : 609.44
0   2 : 493.485
0   3 : 735.466
1   0 : 0.0787317
1   1 : 0.92641
Total runtime: 6.3 s user 0.05 s system 
